TomatoBar/zh-Hans.lproj/Localizable.strings

```diff
@@ -0,0 +1,26 @@
+"IntervalsView.min" = "%d 分";
+"IntervalsView.workIntervalLength.label" = "工作时间:";
+"IntervalsView.shortRestIntervalLength.label" = "小憩时间:";
+"IntervalsView.longRestIntervalLength.label" = "长休息时间:";
+"IntervalsView.longRestIntervalLength.help" = "在完成一组工作后的长休息时长";
+"IntervalsView.workIntervalsInSet.label" = "一组中工作的次数:";
+"IntervalsView.workIntervalsInSet.help" = "每组在长时间休息后的工作次数";
+"SettingsView.shortcut.label" = "快捷键";
+"SettingsView.stopAfterBreak.label" = "在休息后停止";
+"SettingsView.showTimerInMenuBar.label" = "在菜单栏中显示计时器";
+"SettingsView.launchAtLogin.label" = "登录时启动";
+"SoundsView.isWindupEnabled.label" = "发条声";
+"SoundsView.isDingEnabled.label" = "叮";
+"SoundsView.isTickingEnabled.label" = "滴答声";
+"TBPopoverView.start.label" = "开始";
+"TBPopoverView.stop.label" = "停止";
+"TBPopoverView.intervals.label" = "时长";
+"TBPopoverView.settings.label" = "设置";
+"TBPopoverView.sounds.label" = "声音";
+"TBPopoverView.about.label" = "关于";
+"TBPopoverView.quit.label" = "退出";
+"TBTimer.onRestStart.title" = "时间到";
+"TBTimer.onRestStart.short.body" = "小憩时间到！";
+"TBTimer.onRestStart.long.body" = "长休息时间到！";
+"TBTimer.onRestFinish.title" = "休息结束";
+"TBTimer.onRestFinish.body" = "继续努力！";
```

Comment:
Can you remove trailing spaces?

Reply:
You mean the space after the exclamation mark? The ！ in chinese takes up two space while the English one ! takes up one only, so if you would like me to use the English one I'll change it

Reply:
Hi, just resolved it